<launch>
  <!--<node name="logitech_gamepad_F710" pkg="joy" type="joy_node"/>-->
  <node name="roboteq_hdc2460" pkg="roboteq_controller" type="hdc2460.py"/>
</launch>
